src/service-worker/core.js

```diff
@@ -71,6 +82,16 @@ function isCdnJsFile(url) {
   );
 }
 
+/**
+ * Determines if a AMP version is blacklisted.
+ * @param {string} version
+ * @return {boolean}
+ */
+function isBlacklisted(version) {
+  // Trim the RTV perfix.
+  version = version.substr(2);
```

Comment:
But what about in 270 years when we get to 14 digit timestamps?! We're depending on the prefix-timestamp to [detect](https://github.com/jridgewell/amphtml/blob/1a4dd238fb8f2be56e7f1e9d9dc7579d6c21abd0/src/service-worker/core.js#L46) the amp version anyways.